websites/W/watch.lonelil.com/metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "393773835620712461",
+		"name": "yourelosingme"
+	},
+	"service": "watch.lonelil.com",
+	"description": {
+		"en": "watch.lonelil.com is your one-stop destination for free movies, anime, TV shows, and Live TV. With a vast collection of content and easy-to-use interface, you'll never run out of entertainment options."
+	},
+	"url": "watch.lonelil.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/Niuxx4z.png",
```

Comment:
Please use 512x512 for logo and largeImageKey/smallImageKeys if possible. Thumbnail looks good